content/en/blogs/value-of-kubesphere.md

```diff
@@ -0,0 +1,75 @@
+---
+title: 'A Container Platform: What is the Value of KubeSphere'
+tag: 'KubeSphere, Introduction'
+createTime: '2020-04-10'
+author: 'Feynman, Ray'
+snapshot: ''
+---
+
+## A Container Platform: What is the Value of KubeSphere
+
+As **a promising newcomer** of the cloud native family, KubeSphere has gained widespread recognition among its users and developers since it joined the open source community nearly two years ago. This article illustrates the position and value of KubeSphere from scratch in a straightforward way and sheds light on why different teams have chosen KubeSphere.
+
+## What does KubeSphere mean for enterprises
+
+KubeSphere is a **multi-tenant** container platform built on Kubernetes with applications at its core. It is capable of full stack IT automated operation and maintenance, streamlining the DevOps workflow for enterprises. KubeSphere not only helps enterprises quickly establish a Kubernetes cluster in public clouds or private data centers, but also provides a set of multi-functional wizard interfaces.
+
+KubeSphere enables enterprises to build a feature rich container cloud platform equipped with the great flexibility and agile deployment of Kubernetes. The platform also offers outstanding solutions to storage, network and stability, creating the same user experience as IaaS. For example, KubeSphere 2.1.1 supports the integration of Alibaba Cloud and Tencent Cloud block storage plugin. In this version, public cloud storage can be mounted to Pod, providing more stable and persistent storage for stateful applications.
+
+![对于企业 KubeSphere 是什么](https://pek3b.qingstor.com/kubesphere-docs/png/20200410133408.png)
+
+In daily operation, maintenance and development, we might need to utilize and manage a wealth of open source tools, while frequently switching between the GUI and CLI of different tools. Users need to be trained for the installation, usage, operation and maintenance of every single tool, while KubeSphere offers a coordinated solution to the management of all these tools, creating a consistent user experience. That means the production efficiency of development, operation and maintenance teams within an enterprise can be greatly improved as we are freed from multiple threads of execution between control panels and command line terminals of various open source components.
+
+![统一纳管工具](https://pek3b.qingstor.com/kubesphere-docs/png/20200410133506.png)
```

Comment:
Same, use English name, this can be searched in search engine like Google or Baidu